client/state/action-types.ts

```diff
@@ -822,6 +822,9 @@ export const REWIND_SIZE_SET = 'REWIND_SIZE_SET';
 export const REWIND_STATE_REQUEST = 'REWIND_STATE_REQUEST';
 export const REWIND_STATE_UPDATE = 'REWIND_STATE_UPDATE';
 export const REWIND_STORAGE_USAGE_LEVEL_SET = 'REWIND_STORAGE_USAGE_LEVEL_SET';
+export const REWIND_BACKUP_RETENTION_UPDATE = 'REWIND_BACKUP_RETENTION_UPDATE';
+export const REWIND_BACKUP_RETENTION_UPDATE_ERROR = 'REWIND_BACKUP_RETENTION_UPDATE_ERROR';
+export const REWIND_BACKUP_RETENTION_UPDATE_SUCCESS = 'REWIND_BACKUP_RETENTION_UPDATE_SUCCESS';
```

Comment:
I think it would be better to not introduce `rewind` to our codebase.
Maybe we can use `JETPACK_BACKUP_` instead of `REWIND_BACKUP_`. What do you think?

In addition, let's keep in mind that we should maintain the action list keep this list alphabetized.